pkg/controller/defaults/memory.go

```diff
@@ -54,6 +55,17 @@ func addDefaultMemory(req router.Request, cfg *apiv1.Config, appInstance *v1.App
 		return err
 	}
 
+	// Remove any memory defaults for containers that are no longer defined in the app.
+	allContainers := appInstance.GetAllContainerNames()
+	for containerName := range appInstance.Status.Defaults.Memory {
```

Comment:
This will also remove it from resolvedOffering, right?

Reply:
This removes it from defaults. There is another (very similar) piece of code that removes it from resolved offerings

Reply:
Oh I missed that, looks good!